automation/test.sh

```diff
@@ -106,10 +106,17 @@ while [ -n "$(kubectl get pods --no-headers | grep -v Running)" ]; do
     sleep 10
 done
 
-# Make sure all containers are ready
-while [ -n "$(kubectl get pods -o'custom-columns=status:status.containerStatuses[*].ready' --no-headers | grep false)" ]; do
+# Make sure all containers except virt-controller are ready
+while [ -n "$(kubectl get pods -o'custom-columns=status:status.containerStatuses[*].ready,metadata:metadata.name' --no-headers | awk '!/virt-controller/ && /false/')" ]; do
```

Comment:
This command isn't working for me. 

```
$ kubectl get pods -o'custom-columns=status:status.containerStatuses[*].ready,metadata:metadata.name' --no-headers
error: containerStatuses is not found
```

Reply:
@davidvossel It is very strange because it just passed Jenkins tests and it also worked for me locally, can you give output of `kubectl get pods`

My output
```
./cluster/kubectl.sh get pods -o'custom-columns=status:status.containerStatuses[*].ready,metadata:metadata.name' --no-headers
false         haproxy-574c8d574-b2f24
true          haproxy-574c8d574-dbz66
false         iscsi-auth-demo-target-tgtd-6d76d6498d-c75xx
true          iscsi-auth-demo-target-tgtd-6d76d6498d-md2gw
false         iscsi-demo-target-tgtd-65f5dcf6c6-47k6t
true          iscsi-demo-target-tgtd-65f5dcf6c6-s5dpn
true          kubevirt-cockpit-demo-75867f467d-fwcml
true,true     libvirt-r446k
false         spice-proxy-9f4649f9b-cwj4d
true          spice-proxy-9f4649f9b-xdclz
true          virt-api-54fdcfcb4f-hw5xt
true          virt-controller-6f55bf764c-6scz2
false         virt-controller-6f55bf764c-g4x9x
true          virt-handler-rqqh2
true,true     virt-manifest-564cd555d5-bxztw
false,false   virt-manifest-564cd555d5-lrztm
```

Reply:
ah, I see. I'm fairly confident something is wrong with my environment now.

Reply:
@davidvossel I saw @vladikr encountered this issue https://github.com/kubernetes/kubernetes/issues/53356, so it can be also your case too

Reply:
I think that there is also a small time window, directly after the pod post, where the container status is not yet there, then you would also see that. The bash script will and should retry in such a case ...